package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "vui-typography",
-  "version": "0.6.0",
+  "version": "0.7.0",
```

Comment:
Ya cool. I was thinking about that last night and was gonna ask you about that.  :)  Will update that.